mitmproxy/contentviews/image/metadata.py

```diff
@@ -1,21 +1,26 @@
 from . import png
 
 def get_png(img):
-    metadata = [('Format', 'Portable network graphics')]
+    # For debugging
+    try:
+        img = png.Png(img)
+    except Exception as e:
+        print(type(e).__name__, e.args)
```

Comment:
The result of this is `AttributeError ("'bytes' object has no attribute 'read'",)`